ibmcloudant/common.py

```diff
@@ -23,25 +23,26 @@
 USER_AGENT_HEADER = 'User-Agent'
 SDK_NAME = 'cloudant-python-sdk'
 
-# pylint: disable=missing-function-docstring
-def get_system_info():
+
+def get_system_info():  # pylint: disable=missing-docstring
```

Comment:
I think it would be good to write these missing docs now. I mean not in this PR. But opening an issue for it.
These modules are quite succinct otherwise: https://ibm.github.io/cloudant-python-sdk/docs/test-doc/ibmcloudant.html#module-ibmcloudant.common
WDYT?

Reply:
Can you please open a ticket for this, @vmatyusGitHub ?

Reply:
Sure, here it is: https://github.com/IBM/cloudant-python-sdk/issues/49

Reply:
Thank you!